form.yml

```diff
@@ -97,23 +179,118 @@ cluster:
       $name: New resource agent
       $optional: true
       $type: text
-
-  configure_checkbox:
-    $name: Configure resource agents with configuration file
-    $type: boolean
-    $default: false
+  
+  corosync:
```

Comment:
Here other thing we have to change. This corosync configuration cannot be hardcoded with these parameters. These are only examples

Reply:
Thanks I have updated it. But for some configurations like `totem` it contains sub-groups or nested key/value pair. I need to find a way to add such nested configurations